controllers/chat/moveToAnon.js

```diff
@@ -0,0 +1,130 @@
+const {User, ChatAnonUserInfo} = require('../../databases/models');
+const {StreamChat} = require('stream-chat');
+const ErrorResponse = require('../../utils/response/ErrorResponse');
+const {responseError, responseSuccess} = require('../../utils/Responses');
+const {CHANNEL_TYPE} = require('../../helpers/constants');
+const UsersFunction = require('../../databases/functions/users');
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `UsersFunction` module is imported but it's not clear if it contains all necessary functions for user operations. It would be better to import only the required functions directly, which can improve readability and maintainability.
```diff
- const UsersFunction = require('../../databases/functions/users');
+ const { findUserById, findMultipleUsersById } = require('../../databases/functions/users');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->